inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp

```diff
@@ -198,19 +252,23 @@ void FrontEnd::parseLSTMCell(const Model& model, const ie::CNNLayerPtr& _layer,
 
     const auto& src = inputs[0]->desc();
 
-    const std::size_t nBatches = src.dim(Dim::N);
+    std::size_t nBatches = src.dim(Dim::N);
```

Comment:
Why not `const`. It is better to declare variable as `const` if they are not supposed to be modified.